missed_events_replayer/missed_events_replayer/check_logs.py

```diff
@@ -0,0 +1,78 @@
+import time
+import re
+from datetime import datetime, timedelta
+
+from fetch_missing_matomo_requests import download_failed_requests
+from helpers import get_date, confirm_or_abort, get_logger
+
+DATETIME_FORMAT = '%m/%d/%Y %H:%M:%S'
+QUERY_STRING = """fields @message
+    | filter @logStream like /matomo-nginx/
+    | filter status >= 500
+    | filter user_agent != 'ELB-HealthChecker/2.0'
+    | filter path like /idsite=1/
+    | filter path like /rec=1/
+    """
+RESULTS_LIMIT = 1
+LOGGER = get_logger()
+
+
+def query_limits(client, start_date, end_date, sort_order):
+    return client.start_query(
+        logGroupName = 'matomo',
+        startTime=int(start_date.timestamp() * 1000),
+        endTime=int((end_date + timedelta(1)).timestamp() * 1000),
```

Comment:
This is everywhere! Sometimes timestamps are in seconds since the epoch, and sometimes they're in miliseconds since the epoch. 🤷‍♂️ Sorry, just experiencing and sharing frustration at it, not actually a comment on your code here - which I'm sure does the right thing.